VSCode_format_guide.md

```diff
@@ -0,0 +1,20 @@
+# How to Format to Google Java Style using VSCODE:
+1. Install VSCode
+2. Open any Java file with VSCODE
+3. Install the suggested Java plugins for VSCODE. It's called "Java Extension pack". (contains: Language Support for Java™ by Red Hat; Debugger for Java; Java Test Runner;Maven for Java; Java Dependency Viewer; Visual Studio IntelliCode)
+4. Go to settings. Search for "format". Scroll down until you find "settings.json" and open it and add the following lines:
```

Comment:
I think it would be beneficial for the reader if we used markdown formatting for the `format` and `settings.json` options.